src/nupic/algorithms/Connections.hpp

```diff
@@ -221,6 +221,8 @@ namespace nupic
       class Connections
       {
       public:
+        static const UInt VERSION = 1;
```

Comment:
@scott Lifted this from Cells4 and, removed the class var and updated save & load.